cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/CassandraCradleStorage.java

```diff
@@ -91,6 +91,7 @@ public class CassandraCradleStorage extends CradleStorage
 	private MessagesWorker messagesWorker;
 	private BookCache bookCache;
 	private StatisticsWorker statisticsWorker;
+	private EventBatchDurationWorker eventBatchDurationWorker;
```

Comment:
why not ditch EventBatchDurationWorker and move everything from there to EventsWorker?
I do not see strong reason to having separate "worker" for it